packages/terra-application/docs/README.md

```diff
@@ -0,0 +1,285 @@
+# Terra Application
+
+The `terra-application` package provides a baseline of functionality for applications built with Terra components.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-application`
+
+## Usage
+
+The `terra-application` package should be used both by applications and libraries to define a baseline of expected features and functionality for their components.
+
+For application packages:
+
+1. `terra-application` must be specified as a dependency.
+2. A single `<Application />` component must be rendered as high as possible in the component tree.
+3. Components provided by `terra-application` should be consumed solely through `terra-application` and not through their own separate packages.
+
+For library packages:
+
+1. `terra-application` must be specified as a peerDependency and devDependency.
+2. Components within the library should be implemented with the assumption that they will be rendered within an `<Application />` and thus able to access its i18n, theme, and breakpoint APIs.
+3. Components provided by `terra-application` should be consumed solely through `terra-application` and not through their own separate packages.
+
+### Application
+
+The `<Application />` component itself is an aggregate of a few different components, namely `terra-base`, `terra-theme-provider`, and `terra-breakpoints`. When a component is wrapped in an `Application`, it is being wrapped by `Base`, `ThemeProvider`, and `ActiveBreakpointProvider`.
+
+```jsx
+import Application from 'terra-application';
+
+const Example = () => (
+  <Application
+    locale="en-US"
+  >
+    <ExampleContent />
+  </Application>
+)
+```
+
+Only a single `<Application />` component should be rendered within a given application. All Terra components should be rendered within the `<Application />` component to ensure they have access to the framework features.
```

Comment:
nit: should be rendered within a given **_React_** application
I think updating to this as many React applications may live inside of one of our Rails applications.

Reply:
👍 i've updated this a bit